core/server/index.js

```diff
@@ -162,15 +162,15 @@ function init(options) {
             return Promise.reject(response.error);
         }
     }).then(function () {
+        // Initialize the permissions actions and objects
+        // NOTE: Must be done before initDbHashAndFirstRun calls
+        return permissions.init();
```

Comment:
The new call to `settings.read()` in `api.init()` requires that `permissions.init()` already be called, so these have been swapped over.